README.md

```diff
@@ -258,21 +258,590 @@ As part of the deployment, you'll need to set up [git integration](https://rasa.
+
+#### AWS IAM User API Keys:
+
+To allow the GitHub actions to configure the aws cli, create IAM User API Keys as described below, and add them as GitHub Secrets to the repo:
+
+- AWS_ACCESS_KEY_ID = `Access key ID`
+- AWS_SECRET_ACCESS_KEY = `Secret access key` 
+
+#### AWS Elastic IP:
+
+Create an Elastic IP as described below, and add it as a GitHub Secret to the repo:
+
+- AWS_ELASTIC_IP = `PublicIp`
+
+#### Rasa Enterprise License:
+
+To allow the GitHub actions to define a pull secret for the private GCR repo, get the private values from your Rasa Enterprise license file ([docs](https://rasa.com/docs/rasa-x/installation-and-setup/install/helm-chart#5-configure-rasa-x-image)) and add them as GitHub Secrets to the repo:
+
+- GCR_AUTH_JSON_PRIVATE_KEY_ID = `private_key_id`
+- GCR_AUTH_JSON_PRIVATE_KEY = `private_key`
+- GCR_AUTH_JSON_CLIENT_EMAIL = `client_email`
+- GCR_AUTH_JSON_CLIENT_ID = `client_id`
+
+#### Helm chart Credentials
+
+To allow the GitHub actions to use safe_credentials in the `values.yml` ([docs](https://rasa.com/docs/rasa-x/installation-and-setup/install/helm-chart#3-configure-credentials)), add following GitHub Secrets to the repo, replacing each `<safe credential>` with a different alphanumeric string, and choose a `<username>` for the initial user. 
+
+*(Please use **safe credentials** to avoid data breaches)*
+
+- GLOBAL_POSTGRESQL_POSTGRESQLPASSWORD = `<safe credential>`
+- GLOBAL_REDIS_PASSWORD = `<safe credential>`
+- RABBITMQ_RABBITMQ_PASSWORD = `<safe credential>`
+- RASAX_INITIALUSER_USERNAME = `<username>`
+- RASAX_INITIALUSER_PASSWORD = `<safe credential>`
+- RASAX_JWTSECRET = `<safe credential>`
+- RASAX_PASSWORDSALT = `<safe credential>`
+- GLOBAL_POSTGRESQL_POSTGRESQLPASSWORD = `<safe credential>`
+- GLOBAL_POSTGRESQL_POSTGRESQLPASSWORD = `<safe credential>`
+
+
+
+## AWS Preparation
+
+The CI/CD pipeline of financial-demo uses AWS for all the storage & compute resources.
+
+After cloning or forking the financial-demo GitHub repository you must set up the following items before the pipeline can run.
+
+### IAM User API Keys
+
+The CI/CD pipeline uses the [aws cli](https://docs.aws.amazon.com/cli/latest/userguide/install-cliv2-linux.html#cliv2-linux-install).
+
+The [aws cli](https://docs.aws.amazon.com/cli/latest/userguide/install-cliv2-linux.html#cliv2-linux-install) needs a set of IAM User API keys for authentication & authorization:
+
+- In your AWS Console, go to IAM dashboard to create a new set of API keys:
+
+  - Click on Users
+
+  - Click on Add user
+
+    - User name = findemo  *(The actual name is not important, we will never use this name directly)*
+
+      Choose "**programmatic access**." This allows you to use the aws cli to interact with AWS.
+
+    - Click on Next: Permissions
+
+      - Click on *Attach existing policies directly*
+
+        For IAM access, you can choose “**AdministratorAccess**”, or limit access to only what is needed by the CD pipeline.
```

Comment:
Do you know what specific permissions would be required in AWS here?

Reply:
That will require some trial & error I think.
What if we create an issue for this and table that investigation for after the merge.
I agree it would be very helpful to have that information.l

Reply:
I created issue #121